lib/OpenQA/Worker/Engines/isotovideo.pm

```diff
@@ -187,7 +187,7 @@ sub engine_workit {
     session->on(
         collected_orphan => sub {
             my ($session, $p) = @_;
-            log_info("Collected unknown process with pid " . $p->pid . " and exit status: " . $p->exit_status,
+            log_debug("Collected unknown process with pid " . $p->pid . " and exit status: " . $p->exit_status,
```

Comment:
I don't understand, when the log messages are "annoying" as you state, why make them more verbose?

Reply:
This was the solution recommended by @Martchus. To me the whole event doesn't make much sense, but it seems there is nobody left on the team who knows why this was turned into a log message in the first place... so not just outright removing it seems like the safest choice.

Reply:
@okurz Why would that change make it more verbose? It in fact will silence the message unless debug messages are turned on. But we can also get completely rid of it and only log an error in case of non-zero exist status.

Reply:
And if that event doesn't make sense at all (e.g. because a failure to run a subprocess would be logged elsewhere anyways) we should really just remove it.

Reply:
Making it a debug message will not help to keep autoinst-log.txt clean which is already an unreadable mess. Now it looks like this:

```
[2019-09-02T12:49:26.0182 CEST] [info] [pid:18157] Collected unknown process with pid 27786 and exit status: 0
[2019-09-02T12:49:26.643 CEST] [debug] no match: 268.9s, best candidate: pkcon-finished-20180704 (0.00)
[2019-09-02T12:49:27.0271 CEST] [info] [pid:18157] Collected unknown process with pid 27787 and exit status: 0
[2019-09-02T12:49:27.645 CEST] [debug] no match: 267.9s, best candidate: pkcon-finished-20180704 (0.00)
[2019-09-02T12:49:28.0371 CEST] [info] [pid:18157] Collected unknown process with pid 27788 and exit status: 0
[2019-09-02T12:49:28.620 CEST] [debug] no change: 266.9s
[2019-09-02T12:49:29.0454 CEST] [info] [pid:18157] Collected unknown process with pid 27789 and exit status: 0
[2019-09-02T12:49:29.852 CEST] [debug] no match: 265.9s, best candidate: pkcon-finished-20180704 (0.00)
[2019-09-02T12:49:30.0568 CEST] [info] [pid:18157] Collected unknown process with pid 27790 and exit status: 0
[2019-09-02T12:49:30.620 CEST] [debug] no change: 264.9s
[2019-09-02T12:49:31.576 CEST] [debug] no change: 264.0s
[2019-09-02T12:49:31.0632 CEST] [info] [pid:18157] Collected unknown process with pid 27791 and exit status: 0
[2019-09-02T12:49:32.577 CEST] [debug] no change: 263.0s
[2019-09-02T12:49:32.0687 CEST] [info] [pid:18157] Collected unknown process with pid 27792 and exit status: 0
[2019-09-02T12:49:33.577 CEST] [debug] no change: 262.0s
[2019-09-02T12:49:33.0791 CEST] [info] [pid:18157] Collected unknown process with pid 27794 and exit status: 0
[2019-09-02T12:49:34.578 CEST] [debug] no change: 261.0s
[2019-09-02T12:49:34.0856 CEST] [info] [pid:18157] Collected unknown process with pid 27797 and exit status: 0
[2019-09-02T12:49:35.842 CEST] [debug] no match: 260.0s, best candidate: pkcon-finished-20180704 (0.00)
```

So I'd just remove that log line completely.

Reply:
Allright, that's 3 votes for remove then. 😉

Reply:
sure, remove it. Sorry, I got confused: Of course it's not making it "more verbose" but the opposite.